app/models/opponent.rb

```diff
@@ -93,7 +93,7 @@ def method_missing(method, *args)
     end
   end
 
-  def respond_to_missing?(meth, include_private = false)
+  def respond_to_missing?(meth, include_private = false) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
Meta-programming is not my forté, but if the parameter is not used, can't we just delete it?